Firestore/core/test/unit/remote/bloom_filter_test.cc

```diff
@@ -118,7 +118,20 @@ TEST(BloomFilterUnitTest, CreateShouldReturnNotOKStatusIfPaddingIsTooLarge) {
   EXPECT_EQ(maybe_bloom_filter.status().error_message(), "Invalid padding: 8");
 }
 
-TEST(BloomFilterUnitTest, MightContainCanProcessNonStandardCharacters) {
+TEST(BloomFilterTest, CanCheckBloomFiltersEquality) {
```

Comment:
nit: Add a test for `operator!=` as well. This test doesn't need to verify the _entire_ domain of possibilities, like the tests for operator== does. Just do one check when operator!= returns true, and another when it returns false.